tests/utils/dockercli/containers.go

```diff
@@ -86,3 +86,24 @@ func GetVolumeProperties(volumeName, hostName string) (string, error) {
 	return ssh.InvokeCommand(hostName, cmd)
 }
 
+// WriteToContainer write data to a file in the running container
+func WriteToContainer(ip, containerName, volPath, fileName, data string) (string, error) {
```

Comment:
we do have the util in place https://github.com/vmware/docker-volume-vsphere/blob/master/tests/utils/dockercli/volumelifecycle.go#L98

Any reason for introducing this one?